js/main.js

```diff
@@ -1,9 +1,23 @@
 /* global data */
 /* exported data */
-var $image = document.querySelector('.form-image');
+var $formImage = document.querySelector('.form-image');
 var $photoUrl = document.querySelector('#entry-url');
 var $form = document.querySelector('form');
+var $entryList = document.querySelector('.entry-list');
+var $entriesTab = document.querySelector('.entries-tab');
+var $newEntryButton = document.querySelector('.new-entry-button');
+var $entryForm = document.querySelector('.entry-form');
+var $entries = document.querySelector('.entries');
 
+function updateEntryView() {
+  if(data.entries.length <= 0) {  
+    return;
+  }
+  $entryList.innerHTML = '';
```

Comment:
This tactic will come back to bite you if your user ever makes an entry then deletes that entry immediately. You should make an element with text content that you can hide and show as necessary instead